src/uikit.cs

```diff
@@ -13079,6 +13125,10 @@ interface UIScrollView : UIFocusItemScrollableContainer {
 		[Export ("flashScrollIndicators")]
 		void FlashScrollIndicators ();
 
+		[TV (17, 4), iOS (17, 4), MacCatalyst (17, 4)]
+		[Export ("withScrollIndicatorsShownForContentOffsetChanges:")]
+		void WithScrollIndicatorsShownForContentOffsetChanges (Action changes);
```

Comment:
This looks a bit better IMHO:

```suggestion
		void WithScrollIndicatorsShown (Action forContentOffsetChanges);
```